crates/server/src/server/server_future.rs

```diff
@@ -23,6 +23,9 @@ use tokio::{net, task::JoinSet};
 use tokio_util::sync::CancellationToken;
 use tracing::{debug, info, warn};
 
+#[cfg(any(feature = "dns-over-openssl", feature = "dns-over-rustls"))]
```

Comment:
Nit: drop the empty line above to make keep this into the block of third-party crate imports.